### PR TITLE
editor: Right-click add command "Select in File Explorer"

### PR DESCRIPTION
**Pull request by jm-observer**
The effect is as follows:

https://github.com/user-attachments/assets/a9dacfc3-1135-4e8b-954f-8307fff6189e

### CODE REVIEW

File: lapce-app/src/command.rs [hunk 1]
```diff
@@ -183,6 +183,10 @@ pub enum LapceWorkbenchCommand {
     #[strum(message = "Open File")]
     OpenFile,
 
+    #[strum(serialize = "select_in_file_explorer")]
+    #[strum(message = "Select in File Explorer")]
+    SelectInFileExplorer,
```
Comment (panekj) [inline lapce-app/src/command.rs:188]: It should be named `RevealInFileTree` or `RevealActiveFileInFileTree`

**Review by panekj — CHANGES_REQUESTED** [review]

File: lapce-app/src/file_explorer/data.rs [hunk 2]
```diff
@@ -399,6 +401,58 @@ impl FileExplorerData {
         }
     }
 
+    pub fn select(&self, path: PathBuf) {
```
Comment (panekj) [inline lapce-app/src/file_explorer/data.rs:404]: this should be `reveal` or something about presenting it in view, `select` has specific meaning in UI

Comment (jm-observer) [reply lapce-app/src/file_explorer/data.rs]: done

File: lapce-app/src/command.rs (hunk 1 above)
Comment (jm-observer) [reply lapce-app/src/command.rs]: done

**Pull request merged**